shub/deploy.py

```diff
@@ -169,7 +170,8 @@ def _get_pipfile_requirements(tmpdir=None):
         from pipenv.utils.indexes import prepare_pip_source_args
     except ImportError:
         try:
-            from pipenv.utils import convert_deps_to_pip, prepare_pip_source_args
+            from pipenv.utils import (convert_deps_to_pip,
+                                      prepare_pip_source_args)
```

Comment:
Please revert this change, we don't have such line length restriction in place.

Reply:
Sorry, it is an addiction that I have to format the code after making any changes. I have already returned it to the previous state.